pkgs/default.nix

```diff
@@ -17801,6 +17801,7 @@ license = stdenv.lib.licenses.mit;
 , vector
 , wai
 , wai-cors
+, wai-middleware-throttle
```

Comment:
This library is [small](https://github.com/creichert/wai-middleware-throttle/blob/master/Network/Wai/Middleware/Throttle.hs), [has a test-suite](https://github.com/creichert/wai-middleware-throttle/blob/master/test/WaiMiddlewareThrottleSpec.hs), is relatively [well-documented](http://hackage.haskell.org/package/wai-middleware-throttle-0.2.2.1/docs/Network-Wai-Middleware-Throttle.html), and is [actively maintained](https://github.com/creichert/wai-middleware-throttle/commits/master).

The [only alternatve on Hackage](http://hackage.haskell.org/package/wai-throttler) has none of these properties.